ui/ui.go

```diff
@@ -85,6 +85,13 @@ func PrintSelected(name, value string, opts ...Option) error {
 	return nil
 }
 
+// Print if the value (eg. a specific flag value) is empty
+func PrintlnIfEmpty(s, value string) {
+	if value == "" {
+		Println(s)
+	}
+}
```

Comment:
I'm not sure if we should add this method here, it doesn't fit the way other methods work, ideally `ui.Println` should be able to detect `ui.Options` and act accordingly, but that's a bit more difficult. 

I would move this method to `command/ca/init.go` and make it private.